charts/kyverno/values.yaml

```diff
@@ -38,3 +38,5 @@ kyverno:
   priorityClassName: csm-high-priority-service
  # Desired number of pods/replica
   replicaCount: 3
+  config:
+    webhooks: [{"namespaceSelector":{"matchExpressions":[{"key":"kubernetes.io/metadata.name","operator":"NotIn","values":["kube-system"]}]}}]
```

Comment:
@jeremy-duckworth mentioned in the JIRA ticket that we need to "ensure that the kyverno namespace exclude is not overridden".   I think we also need to add that namespace to the webhook namespaceSelector configuration.

https://github.com/kyverno/kyverno/blob/main/charts/kyverno/templates/config/configmap.yaml#L35-L41

Reply:
I forgot that we are using kyverno 1.6.2 which is several version off of main.   1.6.2 doesn't use `excludeKyvernoNamespace` value in configmap.yaml and doesn't define it in values.yaml.

https://github.com/kyverno/kyverno/blob/v1.6.2/charts/kyverno/templates/configmap.yaml#L20-L22

So I think that means that the kyverno exclude needs to be specified in `config.webhooks` the same way it is for the `kube-system` exclude.

Reply:
Yes @johren-hpe, Thank you. @pradeepkumargl missed to add this. Hope changing it as below to exclude both.
webhooks: [{"namespaceSelector":{"matchExpressions":[{"key":"kubernetes.io/metadata.name","operator":"NotIn","values":["kyverno","kube-system"]}]}}]

Reply:
OR, We need add something like for better readability?
config:
  webhooks:
  - namespaceSelector:
      matchExpressions:
      - key: kubernetes.io/metadata.name
        operator: NotIn
        values:
        - kyverno
        - kube-system

Reply:
included kyverno along with kube-system and submitted the changes after testing.